src/test/javascript/spec/component/assessment-shared/assessment-header.component.spec.ts

```diff
@@ -384,4 +385,23 @@ describe('AssessmentHeaderComponent', () => {
         expect(spyOnControlShiftAndArrowRight).toHaveBeenCalledOnce();
         expect(nextSpy).toHaveBeenCalledOnce();
     });
+
+    it('should enable saving after entering an internal assessment note', () => {
+        expect(component.saveDisabled).toBeTruthy();
+        component.isAssessor = true;
+        component.result = new Result();
+        component.result.assessmentNote = new AssessmentNote();
+        expect(component.saveDisabled).toBeTruthy();
+        component.result.assessmentNote.note = 'some input';
+        expect(component.saveDisabled).toBeFalsy();
```

Comment:
Would probably have done it that way, if IntelliJ would not tell me very aggressively that `TS2339: Property 'toBeTrue' does not exist on type 'JestMatchers '.`. It works anyway, though, so I will the more specific ones.

Reply:
The IDE probably does not detect it properly because it is part of the Jest extended library (https://jest-extended.jestcommunity.dev/docs/matchers/boolean/#tobetrue) not base Jest.

Reply:
I see.